### DISCUSSION TITLE
Questions and observations with Apple Cinema Display 30"

### DISCUSSION THREAD

**Discussion by forthrin** [MonitorControl/MonitorControl#1315] [Q&A]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

I have read other issues and discussions, but causes and effects seem so complex that it isn't evident to a non-developer what might constitute a duplicate, so I choose to post my results and questions in full in hope of a helpful response.

Apple Cinema Display 30" (A1083)
- The physical brightness "-" key reduces brightness, but shows no OSD.
- The physical brightness "+" key does not increase brightness.

MonitorControl 4.1.0
- Enable smooth OFF
- Combine HW/SW OFF

Test 1: MacBook Air 2017 with macOS 12.6.3 via Apple Dual Link DVI cable
- Shows "Control method: Hardware (Apple)", which DOES work
- Brightness can also be controlled from System Preferences and Media Keys (without MonitorControl)
- Occasionally, the checkbox once clicked, is immediately disabled (though HW control works!)

Test 2: MacBook Air M1 with macOS 13.2 via CAC-1510-A adapter
- Shows "Control method: Hardware (DDC)", but HW control does NOT work
- Brightness can NOT be controlled from System Preferences and Media Keys (without MonitorControl)
- Occasionally, a volume slider is shown in the Menu bar dropdown, though the monitor has no speakers

Questions, in order of importance:

1. How can I increase the hardware brightness as neither the physical brightness button nor M1/Ventura/MonitorControl can control it. It would be ridiculous to keep the 2017 for the sole purpose of increasing hardware brightness :P

2. What is the common reason the physical brightness buttons stop working? Can this be fixed without opening the case?

3. When reducing hardware brightness, a hissing frequency (around 300 Hz) increases in volume from the left side of the monitor (assumedly the PSU). Are all A1083 specimens like this? Can the noise be eliminated by replacing components?

4. Is there any performance/battery hit on using SW brightness, or is this done on the GPU so it has no practical effect?

5. Do these physical brightness keys control the monitor directly, or do they send signals to the computer via USB, that in turn sends the effective control signals back to the monitor?

6. Is it the M1 (HW) or macOS 13.2 (SW) that does not support hardware control of the monitor?

7. Sometimes, if I exit MonitorControl, the software brightness jumps up to a much higher (normal maximum) level, even though the slider was at max before I exited. Is this a bug? Or how does this situation occur in the first place?

**Comment by waydabber** [comment]
I'll move this one over to discussions, hopefully somebody with experience with this particular display will be able to answer the questions!

**Comment by forthrin** [comment]
Bump
